frontend/src/components/Pages/RobotPage/RobotArmMovement.tsx

```diff
@@ -3,8 +3,8 @@ import { useState, useEffect, useRef } from 'react'
 import { BackendAPICaller } from 'api/ApiCaller'
 import { Robot } from 'models/Robot'
 import { useLanguageContext } from 'components/Contexts/LanguageContext'
-import { tokens } from '@equinor/eds-tokens'
 import styled from 'styled-components'
+import { tokens } from '@equinor/eds-tokens'
```

Comment:
Why?

Reply:
why what?

Reply:
ah i see now, it was my vs code that was being stupid, can change it back